public/components/incontext_insight/index.tsx

```diff
@@ -272,7 +271,18 @@ export const IncontextInsight = ({
           <div className="incontextInsightAnchorContent">{target}</div>
         </EuiFlexItem>
         <EuiFlexItem grow={1}>
-          <div className="incontextInsightAnchorIcon">
+          <div
+            className="incontextInsightAnchorIcon"
+            onClick={onAnchorClick}
+            onKeyDown={(e) => {
+              if (e.key === keys.ENTER || e.key === keys.SPACE) {
+                onAnchorClick();
+              }
+            }}
+            tabIndex={0}
+            role="button"
+            aria-label="Show alert details"
```

Comment:
we should remove this label, `IncontextInsight` is a general component, it could be used in different context, alert is one of those.